pkgs/development/python-modules/tensorflow-tensorboard/default.nix

```diff
@@ -3,33 +3,30 @@
 , numpy
 , werkzeug
 , protobuf
+, grpcio
 , markdown
 , futures
 }:
 
-# tensorflow is built from a downloaded wheel, because
-# https://github.com/tensorflow/tensorboard/issues/719
-# blocks buildBazelPackage.
```

Comment:
Hi, Why do you drop the comment ? It is still downloaded as a wheel.

Reply:
I thought this was incorrect due to 'tensorflow' being mentioned instead of 'tensorboard'. Looks like I was wrong.